arangodb/tests/test_e2e.py

```diff
@@ -0,0 +1,21 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from .common import METRICS, OPTIONAL_METRICS
+
+
+@pytest.mark.e2e
+def test_check_ok(dd_agent_check):
+    aggregator = dd_agent_check(rate=True)
+    base_tags = ['endpoint:http://localhost:8529/_admin/metrics/v2']
+    for metric in METRICS:
+        if metric in OPTIONAL_METRICS:
+            aggregator.assert_metric(metric, at_least=0)
+            for tag in base_tags:
```

Comment:
Can we assert the tag outside the if statement? Also are the mode and server tags available?

Reply:
Server ID is only available for cluster mode, although the env is in standalone mode. I can assert for the mode tag though.